taskcluster/config.yml

```diff
@@ -14,6 +14,8 @@ index:
 
 task-priority: low
 
+docker-image-kind: docker-image
```

Comment:
I would lean towards not defining this unless we actually want to use a non-default name, but doesn't really matter I guess.